### PR TITLE
Test to run gdb testsuite - v4

### PR DESCRIPTION
**Pull request by Pavithra1602**
link to v3 - https://github.com/avocado-framework/avocado-misc-tests/pull/113

### CODE REVIEW

File: generic/gdb.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# Author: Pavithra <pavrampu@linux.vnet.ibm.com>
+
+import os
+import re
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import build
+from avocado.utils import distro
+from avocado.utils import process
```
Comment (PraveenPenguin) [inline generic/gdb.py:24]: can not be do like this way
from avocado.utils import distro, archive, process, build

Comment (PraveenPenguin) [inline generic/gdb.py:24]: otherwise looks good to me

Comment (apahim) [inline generic/gdb.py:24]: According to pep8, both are ok: https://www.python.org/dev/peps/pep-0008/#imports

**Comment by apahim** [comment]
Thank you!

**Pull request merged**